src/components/form/form.stories.mdx

```diff
@@ -319,7 +308,7 @@ It is possible to render `Form` as a content of `DialogFullScreen` component.
   </Story>
 </Preview>
 
-With sticky footer
+#### With sticky footer
```

Comment:
why we are using here `####`?

Reply:
Because it is still part of the wider "In Dialog Fullscreen" example. If we wanted to make it ### Then I think the title should be changed to "In dialog fullscreen with sticky footer"